packages/squiggle-lang/scripts/run-project.mjs

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env node
+import fs from "fs";
+
+import { Command } from "commander";
+
+import { SqProject } from "@quri/squiggle-lang";
+import { runProject } from "./lib.mjs";
+
+const program = new Command();
+
+program.option("-o, --output");
+program.arguments("<string>");
+
+const options = program.parse(process.argv);
+
+const sampleCount = process.env.SAMPLE_COUNT ?? 10000;
```

Comment:
In `run-file.mjs` sampleCount is left undefined if it's not set. Also, you didn't pass this var to `runProject` later on, so this doesn't do anything.